packages/patternfly-4/react-catalog-view-extension/src/components/CatalogTile/CatalogTile.tsx

```diff
@@ -59,11 +62,43 @@ export class CatalogTile extends React.Component<CatalogTileProps> {
     footer: null as string | React.ReactNode
   };
 
+  state = {
+    heightStyle: {}
+  };
+
+  descFullHeight = null as any;
+  descLineHeight = null as any;
+  footerFullHeight = null as any;
+  cardFullHeight = null as any;
+  headerFullHeight = null as any;
+  headFullHeight = null as any;
+
+  componentDidMount() {
+    this.computeDescHeight();
+  }
+
+  computeDescHeight() {
+    let heightStyle = { maxHeight: null as string, WebkitLineClamp: null as string };
+    if (this.cardFullHeight && this.headFullHeight && this.headerFullHeight && this.descFullHeight) {
```

Comment:
Could the heights be `0`? If so the computation should still occur.

Reply:
Fixed.